packages/bierzo-wallet/src/logic/connection.ts

```diff
@@ -5,39 +5,38 @@ import { createLiskConnector } from "@iov/lisk";
 
 import { ChainSpec, getConfig } from "../config";
 
-let ethereumConnection: BlockchainConnection | undefined;
-let bnsConnection: BlockchainConnection | undefined;
-let liskConnection: BlockchainConnection | undefined;
+export interface ChainConnections {
+  [chainId: string]: BlockchainConnection;
+}
+
+const connections: ChainConnections = {};
```

Comment:
Can we make this a `Map<string, BlockchainConnection>`? This gives us much better type safety for the getters